app/controllers/api/v2/containers_controller.rb

```diff
@@ -108,7 +108,7 @@ def logs
         render :json => { :logs => Docker::Container.get(@container.uuid)
           .logs(:stdout => (params[:stdout] || true),
                 :stderr => (params[:stderr] || false),
-                :tail   => (params[:tail]   || 100)) }
+                :tail   => (params[:tail] || 100)) }
```

Comment:
I think there is a config option to allow the alignment of operators. Interested to hear what other people say here.